core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -174,3 +199,26 @@ message UpdateEntryResponse {
   // proof contains a proof that the update has been included in the tree.
   GetEntryResponse proof = 1;
 }
+
+// GetMutationsRequest contains the input parameters of the GetMutation APIs.
+message GetMutationsRequest {
+  // page_token defines the starting point for pagination. An empty
+  // value means start from the beginning. A non-empty value is used by
+  // the server as an epoch or a mutation sequence number depending on
+  // the API called.
+  string page_token = 4;
```

Comment:
`page_token` is supposed to be opaque.  Consider "a non-empty value requests the next page of values"